### PR TITLE
fix: [SRM-14383]: Reset monitored service filters on project change

### PR DESCRIPTION
**Pull request by praneshg239**
### Summary

Monitored service list page: 
1. Added search text filter to risk filter query params
2. Resetting page on project change 
3. Reset ENV, search text on project change
4. Reset risk filter

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/monitored-service/CVMonitoredService/components/MonitoredServiceListView/MonitoredServiceListView.tsx [hunk 1]
```diff
@@ -419,7 +418,7 @@ ET_DEPLOYMENT_NAME: <replace with deployment version>`
               itemCount: totalItems,
               gotoPage: nextPage => {
                 setPage(nextPage)
-                refetchServiceCountData()
+                // refetchServiceCountData()
```
Comment (karansaraswat19) [inline src/modules/85-cv/pages/monitored-service/CVMonitoredService/components/MonitoredServiceListView/MonitoredServiceListView.tsx:421]: Why did we comment this?

Comment (praneshg239) [reply src/modules/85-cv/pages/monitored-service/CVMonitoredService/components/MonitoredServiceListView/MonitoredServiceListView.tsx]: Removed

**Review by karansaraswat19 — DISMISSED** [review]

File: src/modules/85-cv/pages/monitored-service/CVMonitoredService/__test__/CVMonitoredServiceFilterReset.test.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+beforeEach(() => jest.clearAllMocks())
+
+jest.spyOn(cvServices, 'useDeleteMonitoredService').mockImplementation(() => ({ mutate: jest.fn() } as any))
+jest.spyOn(cvServices, 'useGetMonitoredServiceListEnvironments').mockImplementation(
+  () =>
+    ({
+      data: {
+        data: [
+          { environment: { accountId: 'kmpySmUISimoRrJL6NL73w', name: 'new_env_test', identifier: 'new_env_test' } }
+          // 'new_env_test', 'AppDTestEnv1', 'AppDTestEnv2'
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/monitored-service/CVMonitoredService/__test__/CVMonitoredServiceFilterReset.test.tsx:72]: remove this

Comment (praneshg239) [reply src/modules/85-cv/pages/monitored-service/CVMonitoredService/__test__/CVMonitoredServiceFilterReset.test.tsx]: Done

**Review by deepesh-ui — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

**Pull request merged**